printer-creality-cr10v3-bltouch.cfg

```diff
@@ -0,0 +1,229 @@
+#   to a non-zero value it must be within the range of z-values in the mesh.
+#   Users that wish to converge to the z homing position should set this to 0.
+#   Default is the average z value of the mesh.
+#split_delta_z: .025
+#   The amount of Z difference (in mm) along a move that will
+#   trigger a split. Default is .025.
+#move_check_distance: 5.0
+#   The distance (in mm) along a move to check for split_delta_z.
+#   This is also the minimum length that a move can be split. Default
+#   is 5.0.
+mesh_pps: 2,2
+#   A comma separated pair of integers (X,Y) defining the number of
+#   points per segment to interpolate in the mesh along each axis. A
+#   "segment" can be defined as the space between each probed
+#   point. The user may enter a single value which will be applied
+#   to both axes.  Default is 2,2.
+algorithm: bicubic
+#   The interpolation algorithm to use. May be either "lagrange"
+#   or "bicubic". This option will not affect 3x3 grids, which
+#   are forced to use lagrange sampling.  Default is lagrange.
+bicubic_tension: .2
+#   When using the bicubic algorithm the tension parameter above
+#   may be applied to change the amount of slope interpolated.
+#   Larger numbers will increase the amount of slope, which
+#   results in more curvature in the mesh. Default is .2.
+relative_reference_index: 24
+#   A point index in the mesh to reference all z values to. Enabling
+#   this parameter produces a mesh relative to the probed z position
+#   at the provided index.
+
+[pause_resume]
+recover_velocity: 50
+
+[filament_switch_sensor fil_runout_sensor]
+pause_on_runout: True
+#   When set to True, a PAUSE will execute immediately after a runout
+#   is detected. Note that if pause_on_runout is False and the
+#   runout_gcode is omitted then runout detection is disabled. Default
+#   is True.
+switch_pin: PE4
+#   The pin on which the switch is connected. This parameter must be
+#   provided.
+#runout_gcode:
+#   A list of G-Code commands to execute after a filament runout is
+#   detected. See docs/Command_Templates.md for G-Code format. If
+#   pause_on_runout is set to True this G-Code will run after the
+#   PAUSE is complete. The default is not to run any G-Code commands.
+#insert_gcode:
+#   A list of G-Code commands to execute after a filament insert is
+#   detected. See docs/Command_Templates.md for G-Code format. The
+#   default is not to run any G-Code commands, which disables insert
+#   detection.
+#event_delay: 3.0
+#   The minimum amount of time in seconds to delay between events.
+#   Events triggered during this time period will be silently
+#   ignored. The default is 3 seconds.
+#pause_delay: 0.5
+#   The amount of time to delay, in seconds, between the pause command
+#   dispatch and execution of the runout_gcode. It may be useful to
+#   increase this delay if OctoPrint exhibits strange pause behavior.
+#   Default is 0.5 seconds.
+
+
+[gcode_macro G29]
+gcode:
+    BED_MESH_CALIBRATE
```

Comment:
Remove

Reply:
Ok, I'll remove, but this is very relevant for compatibility with Marlin GCODE